### ISSUE TITLE
Music plays from phone even when connected to a chromecast

### ISSUE THREAD

**Issue by derRAV3N** [naman14/Timber#350]
Hey.
On a Pixel 2, Android 8.1, Version 1.6 of Timber, I can connect to a Chromecast (Audio), but when I then select to play music, it does play through the phone instead of being streamed to the Chromecast. Here is a screenshot showing the popup of the Chromecast dialog, which shows that no media is selected, while in the background, media is played.
![screenshot_20171223-131518](https://user-images.githubusercontent.com/12349529/34319537-f9810876-e7e4-11e7-9482-5a25f7b67f72.png)

# Steps to Repoduce

1. Open Timber
2. Touch the Chromecast Icon
3. Select the device to stream on
4. Select any Song to play
5. Now the Song is playing locally, the Chromecast is still connected.

It also doesn't work to first play a song and then select to Chromecast to stream to, same issue.

**Comment by naman14** [comment]
Hmm, this is weird. Your device is a chromecast audio? I haven't tested with an audio only device but api should have been the same for both normal chromecast and chromecast audio.

**Comment by derRAV3N** [comment]
Yes, I wanted to use this on the chromecast audio. I also have a Google Home Mini and a normal Chromecast (the round one, but not Ultra), I just tested it with both of them, they both also connect but don't play let me play any media.

Is there any specific format used for chromecast?
